internal/metrics/metricsManager.go

```diff
@@ -242,10 +242,11 @@ func (m *MetricsManager) ReportHostInstallationMetrics(log logrus.FieldLogger, c
 	}
 }
 
-func (m *MetricsManager) handleHostInstallationComplete(log logrus.FieldLogger, clusterVersion string, clusterID strfmt.UUID, roleStr string, installationStageStr string, h *models.Host) {
+func (m *MetricsManager) handleHostInstallationComplete(log logrus.FieldLogger, clusterVersion string, clusterID strfmt.UUID, roleStr string,
+	hwVendor string, hwProduct string, diskType string, installationStageStr string, h *models.Host) {
 	log.Infof("service Logic Cluster Hosts clusterVersion %s, roleStr %s, result %s",
```

Comment:
please add the new parameters also to the log

Reply:
Agreed. Done.